api.go

```diff
@@ -2901,14 +2901,24 @@ func (sc *Client) DeleteEdgeDeployment(corpName, siteName string) error {
 	return err
 }
 
+type CreateOrUpdateEdgeDeploymentServiceBody struct {
+	ActivateVersion *bool `json:"activateVersion,omitempty"` //Activate Fastly VCL service after clone
+	PercentEnabled  int   `json:"percentEnabled,omitempty"`  //Percentage of traffic to send to NGWAF@Edge
+}
+
 // CreateOrUpdateEdgeDeploymentService copies the backends from the Fastly service to the
 // Edge Deployment and pre-configures the Fastly service with an edge dictionary and custom VCL.
-func (sc *Client) CreateOrUpdateEdgeDeploymentService(corpName, siteName, fastlySID string) error {
+func (sc *Client) CreateOrUpdateEdgeDeploymentService(corpName, siteName, fastlySID string, body CreateOrUpdateEdgeDeploymentServiceBody) error {
+	b, err := json.Marshal(body)
```

Comment:
Why do this before the `sc.fastlyKey` check below?

Reply:
Fair point :-)

Reply:
@freeformz thanks -- fixed